src/lib/containers/oauth/OAuthManagement.tsx

```diff
@@ -73,6 +74,7 @@ export const OAuthManagement: React.FunctionComponent = () => {
               <tr key={item.client_id}>
                 <td>{formatDate(moment(item.createdOn))}</td>
                 <td>{formatDate(moment(item.modifiedOn))}</td>
+                <td>{item.client_id}</td>
```

Comment:
Since this is in the table, you can remove the "You can find this within 'Actions'" blurb